pkgs/development/python-modules/vllm/default.nix

```diff
@@ -34,65 +44,61 @@
   rocmSupport ? config.rocmSupport,
   rocmPackages ? { },
   gpuTargets ? [ ],
-}:
+}@args:
```

Comment:
would suggest using let-binding `stdenv` appropriately to avoid the shadowing instead of capturing via `@`

Reply:
that was the original approach see https://github.com/NixOS/nixpkgs/pull/328258#discussion_r1684482396

Reply:
We don't `inherit` because we want to choose between `stdenv` and `cudaPackages.backendStdenv`